src/test/resources/db/schema/schema.sql

```diff
@@ -1,237 +1,221 @@
+                                         InternalId int not null,
+                                         Careof varchar(255),
+                                         City varchar(255),
+                                         Company nvarchar(255),
+                                         FacilityId varchar(255),
+                                         HouseName varchar(255),
+                                         Postcode varchar(255),
+                                         Street varchar(255),
+                                         Type nvarchar(255),
+                                         primary key (BillLocationID)
 );
 
-create table kundinfo.vInstalledBase
-(
-    BillLocationID               int not null,
-    DateFrom                     datetime,
-    DateTo                       datetime,
-    InstalledBaseLastChangedDate datetime,
-    InternalId                   int not null,
-    customerid                   int not null,
-    CareOf                       varchar(255),
-    City                         varchar(255),
-    Company                      nvarchar(255),
-    FacilityId                   varchar(255),
-    HouseName                    varchar(255),
-    Postcode                     varchar(255),
-    Street                       varchar(255),
-    Type                         nvarchar(255),
-    primary key (BillLocationID)
+create table kundinfo.vInstalledBase (
+                                         BillLocationID int not null,
+                                         DateFrom datetime,
+                                         DateTo datetime,
+                                         InstalledBaseLastChangedDate datetime,
+                                         InternalId int not null,
+                                         customerid int not null,
+                                         CareOf varchar(255),
+                                         City varchar(255),
+                                         Company nvarchar(255),
+                                         FacilityId varchar(255),
+                                         HouseName varchar(255),
+                                         Postcode varchar(255),
+                                         Street varchar(255),
+                                         Type nvarchar(255),
+                                         primary key (BillLocationID)
 );
 
-create table kundinfo.vInstalledBaseMetadata
-(
-    internalId  int         not null,
-    company     nvarchar(255),
-    displayName varchar(26) not null, [
-    key]
-    varchar
-(
-    255
-) not null, [type] varchar(8) not null, [value] nvarchar(255)
+create table kundinfo.vInstalledBaseMetadata (
+                                                 internalId int not null,
+                                                 company nvarchar(255),
+                                                 displayName varchar(26) not null,
```

Comment:
Indentation seems off here (do we need 50 spaces as indentation b.t.w.?)

Reply:
This is just auto-formatted with intellij. Is it really an issue?

Reply:
Fix your intelliJ settings and reformat the file then. 🚀